places/berlin.ini

```diff
@@ -111,3 +111,28 @@ url_path = div/div/div[2]/h2/a/@href
 title_path = div/div/div[2]/h2/a
 location_path = div/div/div[2]/address:(?<=,)[^,]*
 rooms_path = div/div/div[2]/ul/li[1]/span[2]
+
+[company:www.blp-management.com]
+url = https://blp-management.com/immobilien/?post_type=immomakler_object&vermarktungsart=miete&nutzungsart=wohnen&typ=wohnung&ort=berlin&center&objekt-id&collapse=in
+ad_path = .//div[@class='property-container']
+url_path = div[2]/h3/a/@href
+title_path = div[2]/h3/a
+location_path = div[2]/div
+rooms_path = div[2]/div[2]/div[2]
+
+[company:www.immonexxt.com]
+url = https://www.immonexxt.com/de/mietangebote-wohnen#filter=.geo-ort-berlin
+ad_path = .//div[@id='estate_list']/a
+url_path = ./@href
+title_path = div/div[2]/h2
+location_path = div/div[2]/ul/li[2]/div[2]/span[1]
+rooms_path = div/div[2]/ul/li[5]/div[2]
+
+[company:www.harry-gerlach.de]
+url = https://www.harry-gerlach.de/wohnung-mieten-berlin/
+ad_path = .//div[@class='card__inner--content']
+url_path = div[4]/a/@href
+title_path = div[2]/h2
+location_path = div[1]/p
+rooms_path = div[3]/dl/dd[4]
```

Comment:
Room parsing here doesn't work because it's stored as "3 Zimmer", so I just ignored it for now and set optional = true

Reply:
Your path here works like a charm and "3 Zimmer" is actually parsed as 3. :blush: Optional make sense here nevertheless, because some ads don't have the number of rooms specified.